dotCMS/src/main/java/com/dotmarketing/business/IdentifierFactoryImpl.java

```diff
@@ -527,6 +528,7 @@ protected void deleteIdentifier(Identifier ident) throws DotDataException {
 	}
 
 	@Override
+	@CloseDBIfOpened
```

Comment:
Let's not include the annotation at this level, because the only caller (API) has it already.